4/99_hw/search_server.go

```diff
@@ -0,0 +1,202 @@
+package main
+
+import (
+	"encoding/xml"
+	"fmt"
+	"net/http"
+	"os"
+	"sort"
+	"strconv"
+	"strings"
+)
+
+type Record struct {
+	Id        int    `xml:"id"`
```

Comment:
```suggestion
	ID        int    `xml:"id"`
```

https://go.dev/wiki/CodeReviewComments#initialisms